tools/sgen/output/FluentUIStyle.generated.swift

```diff
@@ -735,214 +855,854 @@ extension FluentUIThemeManagerTheming {
 		}
 
 
-		//MARK: - Elevation
-		public var _Elevation: ElevationAppearanceProxy?
-		open func ElevationStyle() -> ElevationAppearanceProxy {
-			if let override = _Elevation { return override }
-				return ElevationAppearanceProxy(proxy: mainProxy)
+		//MARK: - Brass
+		public var _Brass: BrassAppearanceProxy?
+		open func BrassStyle() -> BrassAppearanceProxy {
+			if let override = _Brass { return override }
+				return BrassAppearanceProxy(proxy: mainProxy)
 			}
-		public var Elevation: ElevationAppearanceProxy {
-			get { return self.ElevationStyle() }
-			set { _Elevation = newValue }
+		public var Brass: BrassAppearanceProxy {
+			get { return self.BrassStyle() }
+			set { _Brass = newValue }
 		}
-		@objc(ColorsElevationAppearanceProxy) @objcMembers open class ElevationAppearanceProxy: NSObject {
+		@objc(ColorsBrassAppearanceProxy) @objcMembers open class BrassAppearanceProxy: NSObject {
 			public let mainProxy: () -> FluentUIStyle
 			public init(proxy: @escaping () -> FluentUIStyle) {
 				self.mainProxy = proxy
 			}
 
-			//MARK: highElevation 
-			public var _highElevation: UIColor?
-			open func highElevationProperty(_ traitCollection: UITraitCollection? = UIScreen.main.traitCollection) -> UIColor {
-				if let override = _highElevation { return override }
-					return UIColor(light: mainProxy().Colors.Shadow.opacity40Property(traitCollection), lightHighContrast: nil, lightElevated: nil, lightElevatedHighContrast: nil, dark: mainProxy().Colors.Shadow.opacity60Property(traitCollection), darkHighContrast: nil, darkElevated: nil, darkElevatedHighContrast: nil)
+			//MARK: shade30 
+			public var _shade30: UIColor?
+			open func shade30Property(_ traitCollection: UITraitCollection? = UIScreen.main.traitCollection) -> UIColor {
+				if let override = _shade30 { return override }
```

Comment:
`_ traitCollection: UITraitCollection? = UIScreen.main.traitCollection` would you please remind me why this is generated?

Reply:
The YAML generator supports adding conditions in the YAML file that are based on the content size.
We're not using any of it, but it is supported by SGen/Stardust.

Reply:
is there  a todo to get rid of that code?

Reply:
Just created a task for that in our ADO Vnext backlog.